politeiawww/api/ticketvote/v1/v1.go

```diff
@@ -35,7 +35,8 @@ const (
 	ErrorCodeRecordLocked     ErrorCodeT = 5
 	ErrorCodeTokenInvalid     ErrorCodeT = 6
 	ErrorCodePageSizeExceeded ErrorCodeT = 7
-	ErrorCodeLast             ErrorCodeT = 8
+	ErrorCodeDuplicatePayload ErrorCodeT = 8
```

Comment:
Can you add a comment to this similar to what you did for politeiad?  I realize the other errors don't have comments, but that's just because I was rushing to get the tstore release shipped.  I'm going to go back at some point and add them.

Reply:
Sure thing, done 👍